modules/sharedaddy/sharing-service.php

```diff
@@ -748,8 +748,17 @@ function sharing_display( $text = '', $echo = false ) {
 		$show = false;
 	}
 
-	// Allow to be used on P2 ajax requests for latest posts.
-	if ( defined( 'DOING_AJAX' ) && DOING_AJAX && isset( $_REQUEST['action'] ) && 'get_latest_posts' == $_REQUEST['action'] ) {
+	/**
+	 * Filter the default ajax action name
```

Comment:
I would suggest being more verbose here, so the message can be really helpful for those stumbling across the filter. You want to let them know exactly what the filter can be used for.

```suggestion
	 * Filter the Sharing buttons' Ajax action name Jetpack checks for, to allow the use of the buttons with your own Ajax implementation.
```